src/v/raft/tests/raft_fixture.cc

```diff
@@ -256,8 +256,8 @@ static constexpr msg_type map_msg_type() {
 }
 
 template<typename ReqT, typename RespT>
-ss::future<result<RespT>>
-in_memory_test_protocol::dispatch(model::node_id id, ReqT req) {
+ss::future<result<RespT>> in_memory_test_protocol::dispatch(
+  model::node_id id, ReqT req, rpc::client_opts opts) {
     _gate.hold();
```

Comment:
Out of scope, but authored by you as well, so I'm curious to ask what that means. It's probably equivalent to `_gate.check()`. TBH I'd mark`ss::gate::hold()` as `[[nodiscard]]`.

Reply:
that should be assigned

Reply:
fixed